spec/controllers/chapters_controller_spec.rb

```diff
@@ -680,8 +680,7 @@
           end
 
           it "posts the work if the work was not posted before" do
-            pending "multi-chapter works should post when chapter is posted"
-            put :update, params: { work_id: unposted_work.id, id: unposted_work.chapters.first.id, chapter: @chapter_attributes, post_button: true }
+            put :update, work_id: unposted_work.id, id: unposted_work.chapters.first.id, chapter: @chapter_attributes, post_button: true
```

Comment:
It looks like this is failing on Travis:
```
 1) ChaptersController update when work owner is logged in when the post button is clicked when the chapter and work are valid posts the work if the work was not posted before
     Failure/Error: put :update, work_id: unposted_work.id, id: unposted_work.chapters.first.id, chapter: @chapter_attributes, post_button: true
     
     ArgumentError:
       unknown keywords: work_id, id, chapter, post_button
```
However, I think it's just a case of needing to put `params: { }` around your parameters.